Lib/test/test_dis.py

```diff
@@ -219,6 +219,45 @@ def bug42562():
           8 RETURN_VALUE
 """
 
+# Result of compile("while not (a < b < c):\n    pass", "", "exec").co_code
+# but added as constant to make agnostic to compile behavior
+bug46724 = b'\x97\x00e\x00e\x01c\x02x\x02k\x00r\x0be\x02k\x00s\x1dn\x01\x01\x00\t\x00e\x00e\x01c\x02x\x02k\x00r\x18e\x02k\x00r\x0cd\x00S\x00\x01\x00\x90\xff\x90\xff\x90\xffn\xefd\x00S\x00'
```

Comment:
You don't need all this code now.
Just
```
     EXTENDED_ARG           255
     EXTENDED_ARG           65535
     EXTENDED_ARG           167...
     JUMP_FORWARD           -4 (to 0)
```
will do.

Reply:
How does it look now?